src/components/filterBox/tests/FilterBox.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import { shallow, mount, ReactWrapper } from 'enzyme';
+import { FilterBox, IFilterBoxProps, FILTER_PLACEHOLDER } from '../FilterBox';
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+describe('FilterBox', () => {
```

Comment:
Instead of removing it, I added it in other files. This way the tests reflect the project structure more.